src/Bundle/test/src/Migration/namespace_grid.yaml

```diff
@@ -0,0 +1,12 @@
+sylius_grid:
+    grids:
+        foo_with_namespace:
+            driver:
+                name: 'doctrine/orm'
+                options:
+                    class: "%sylius.model.order.class%"
+                    pagination:
+                        fetch_join_collection: false
+                        use_output_walkers: false
+            removals: []
```

Comment:
I think that is important to add a test with removals, even if this is brain new

Reply:
I haven't really found out how they would be structured in the yaml. Would they just be an array of fields and actions to remove? If so how would I know which is which?

Reply:
Yes indeed, they are not in the doc... we have to add it into the docs. it's just an array of field keys.